common/thorhelper/thorsoapcall.cpp

```diff
@@ -1524,6 +1524,12 @@ class CWSCAsyncFor : implements IWSCAsyncFor, public CInterface, public CAsyncFo
         if (!httpHeaderBlockContainsHeader(httpheaders, ACCEPT_ENCODING))
             request.appendf("%s: gzip, deflate\r\n", ACCEPT_ENCODING);
 #endif
+        if (!isEmptyString(master->logctx.queryGlobalId()))
+        {
+            request.append("RELX-Global-Id: ").append(master->logctx.queryGlobalId()).append("\r\n");
```

Comment:
This feels rather RELX specific for OSS code. Is there a reason the Global ID would only be useful to RELX?

Reply:
It's more that all RISK components will be using the same headers.  I.e. web applications, scapps.  Now is the time to set the standard, we could do HPCC, but not sure what would fit for all of RISK (and perhaps eventually beyond).

Reply:
What about users of HPCC outside of RELX ? Why would they want to use a global id called "RELX-Global-Id" ?

Reply:
Perhaps something even more generic.  I thought just Global-ID might not be unique enough.

Reply:
I'll bring up a more generic name with the Global ID team.

Reply:
Alternatively, make the name configurable.

Reply:
Yes, I'll do that.